ihmc-communication/src/main/java/us/ihmc/communication/crdt/RequestConfirmFreezable.java

```diff
@@ -84,10 +81,20 @@ public void freeze()
       needToSendRequest.set();
    }
 
-   @Override
-   public void unfreeze()
+   /** internal and tests only */ void unfreeze()
    {
+      boolean wasFrozen = crdtInfo.getUpdateNumber() < updateNumberToUnfreeze;
+
       updateNumberToUnfreeze = crdtInfo.getUpdateNumber();
+
+      isFrozen = crdtInfo.getUpdateNumber() < updateNumberToUnfreeze;
+
+      if (wasFrozen != isFrozen)
+         LogTools.debug("%s Update #%d: Frozen %b -> %b %s".formatted(crdtInfo.getActorDesignation(),
+                                                                      crdtInfo.getUpdateNumber(),
+                                                                      wasFrozen,
+                                                                      isFrozen,
+                                                                      this.getClass().getSimpleName()));
```

Comment:
Now there's always a print when something unfreezes, instead of only some of the time